### PR TITLE
Modify hydra config

### PR DESCRIPTION
**Pull request by Fexiven**
Since theres also the option for 433 Mhz i think this config should be included too

### CODE REVIEW

File: variants/diy/hydra/variant.h [hunk 1]
```diff
@@ -17,7 +17,9 @@
 #define LED_PIN 2           // add status LED (compatible with core-pcb and DIY targets)
 
 // Radio
-#define USE_SX1262 // E22-900M30S uses SX1262
+//#define USE_SX1262 // E22-900M30S uses SX1262 - 868Mhz
+//#define USE_SX1262 // E22-300M30S uses SX1268 - 433Mhz
+
 #define SX126X_MAX_POWER                                                                                                         \
```
Comment (caveman99) [inline variants/diy/hydra/variant.h:23]: Sorry, this change will stop the hydra target precompiled binaries from working. there's no problem defining both chips, as you can see in the t-beam variant.h

**Review by caveman99 — CHANGES_REQUESTED** [review]

File: variants/diy/hydra/variant.h (hunk 1 above)
Comment (Fexiven) [reply variants/diy/hydra/variant.h:23]: I defined both modules then, i wasn't sure about it

**Comment by caveman99** [comment]
That should work fine now, please sign the CLA

**Review by caveman99 — APPROVED** [review]

**Review by jp-bennett — COMMENTED** [review]
Looks good to me. No hardware here to actually test the change.

**Comment by Fexiven** [comment]
I have an tinyVNA but I think that's not suitable for testing so high power modules?

I got an flipper zero, that's not very scientific, but I might see how much dBi it's emitting

I will try later

**Comment by Fexiven** [comment]
Sadly i cant get the module to work outside of meshtastic with minimal config.... Can anyone help? Meshtastic is sadly the only firmware that gets it magically running

**Comment by Fexiven** [comment]
I tried with setting different TX levels... but my Flipper shows -36dbm no matter what i set (1-30)... so i guess my Flipper is incorrect (which is very very likely its not made to be a spectrum analyzer)

i have two of these neat boards, but i don't know how i can get the second board to be a power logger, i don't get the RadioLib to work by myself...

**Pull request closed without merging**